sycl/doc/extensions/GlobalWorkItems/SYCL_INTEL_global_work_items.asciidoc

```diff
@@ -0,0 +1,184 @@
+= SYCL_INTEL_global_work_items
```

Comment:
I think the name here could be clearer.  `SYCL_INTEL_global_work_item_functions`, maybe?

Reply:
Since these are only queries (not other functions), and they don't just expose "work-items" or "global work-items", I'd propose `SYCL_INTEL_free_function_ID_queries` or something along those lines.

Reply:
What about the combination of previous two? something like `SYCL_INTEL_global_ID_queries` (or `SYCL_INTEL_global_ID_queries`)?

Reply:
I prefer @mkinsner's suggestion to my own.  `global` is ambiguous here, and somebody might assume it has something to do with the `global` address space or the `global` range of an ND-range.  `free_function_ID_queries` is more descriptive and unambiguous.

I have a slight preference for `free_function_queries` (without the `ID`) because we already support item and group queries, and I could imagine other queries (e.g. range, device) in future revisions.

Reply:
Done. Agree with John. I don't think that 'ID' adds much value here

Reply:
The name is still showing as `SYCL_INTEL_global_work_items` for me.

Reply:
Looks strange. Could you please check in the "Files changed" tab

Reply:
I see it now.  GitHub or user error. :)  Thanks.